components/NavBar/navbar.module.css

```diff
@@ -1,99 +1,191 @@
-.navBar {
-  display: flex;
-  background-color: #041484;
-  font-size: 0.9rem;
-  width: 100%;
-  /* padding: 0.05rem 0; */
-  position: relative;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: 50px;
+.wrapper {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  margin-bottom: 25px;
 }
-.navBar > ul {
-  display: flex;
-  align-items: center;
-  padding: 0;
-  margin: 0;
+
+.navBar li a {
+  margin: 10px;
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  text-align: center;
+  padding: 19px 16px;
+  text-decoration: none;
 }
+
 .navBar li {
-  list-style-type: none;
-  color: #f2f2f2;
-  font-size: 20px;
-  margin-right: 3rem;
-  height: 100%;
+  float: left;
 }
 
 .navBar li a:hover {
-  color: #e30464;
+  color: #49a82e;
 }
-.logo {
-  margin: 10px 0 5px 10px;
-  cursor: pointer;
+
+.navBarMenu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #1d1283;
 }
-.profilePic {
 
+.navBarLogoLi {
+  padding: 14px 20px;
+}
+
+.navBarLogoLi a {
+  margin: 0px !important;
+  padding: 0px !important;
```

Comment:
Can we avoid using `!important` ?? I see there are many in the file.

Reply:
Yes, removed `!important` property